src/vue/pages/Limits/Limits.RequestDetailsViewer.vue

```diff
@@ -0,0 +1,202 @@
+        {{ 'request-messages.canceled-msg' | globalize() }}
+      </p>
+    </div>
+
+    <div
+      v-else-if="request.isPermanentlyRejected"
+      class="request-state request-state--permanently-rejected"
+    >
+      <p class="request-state__content">
+        <!-- eslint-disable-next-line max-len -->
+        {{ 'request-messages.permanently-rejected-msg' | globalize({ reason: request.rejectReason }) }}
+      </p>
+    </div>
+    <div class="app__table limits-request-details__table">
+      <table>
+        <tbody>
+          <tr>
+            <td>
+              <!-- eslint-disable-next-line -->
+              {{ 'limits-requests-table-renderer.table-request-type-lbl' | globalize }}
+            </td>
+            <!-- eslint-disable-next-line -->
+            <td :title="LIMITS_REQUEST_TYPE_TRANSLATE_ID[request.limitsRequestType] | globalize">
+              <!-- eslint-disable-next-line -->
+              {{ LIMITS_REQUEST_TYPE_TRANSLATE_ID[request.limitsRequestType] | globalize }}
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <!-- eslint-disable-next-line -->
+              {{ 'limits-request-details-viewer.table-daily-limit-lbl' | globalize }}
+            </td>
+            <td>
+              {{ request.limits.dailyOut }} {{ request.asset }}
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <!-- eslint-disable-next-line -->
+              {{ 'limits-request-details-viewer.table-weekly-limit-lbl' | globalize }}
+            </td>
+            <td>
+              {{ request.limits.weeklyOut }} {{ request.asset }}
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <!-- eslint-disable-next-line -->
+              {{ 'limits-request-details-viewer.table-monthly-limit-lbl' | globalize }}
+            </td>
+            <td>
+              {{ request.limits.monthlyOut }} {{ request.asset }}
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <!-- eslint-disable-next-line -->
+              {{ 'limits-request-details-viewer.table-annual-limit-lbl' | globalize }}
+            </td>
+            <td>
+              {{ request.limits.annualOut }} {{ request.asset }}
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="limits-request-details__buttons">
+      <button
+        class="app__button-raised limits-request-details__upload-btn"
+        :disabled="!request.isRejected"
+        @click="$emit(EVENTS.upload)"
+      >
+        {{ 'limits-requests-table-renderer.upload-documents-btn' | globalize }}
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+
+import { LimitsUpdateRequestRecord } from '@/js/records/requests/limits-update.record'
+
+import {
+  LIMITS_REQUEST_STATES_STR,
+  LIMITS_REQUEST_TYPE,
+} from '@/js/const/limits.const'
+
+const LIMITS_REQUEST_STATES_STR_TRANSLATE_ID = Object.freeze({
```

Comment:
`LIMITS_REQUEST_STATES_STR_TRANSLATION_ID`

Reply:
Renamed

Reply:
1. still not renamed in the current file
2. looks like it is unused in the current file

Reply:
My apologies.